templates/cassiopeia/scss/vendor/bootstrap/_custom-forms.scss

```diff
@@ -66,7 +66,7 @@
 }
 
 // Backwards compat with BS4
-.custom-select {
+.form-select {
```

Comment:
Pretty sure this part isn't needed :)

Reply:
My bad :)

Reply:
Maybe I miss-understand... do you want to support the `custom-select` class?

Reply:
Im gonna assume not :)